gotv_transportation.md

```diff
@@ -0,0 +1,169 @@
+| demand_id | string | The Demand to which this Assignment is made
+| pick_up_at_time | datetime | The time at which this assigned ride is estimated to begin
+| pick_up_at_location | Location | The location at which this assigned will begin
+| drop_off_at_time | datetime | The time at which this assigned ride is estimated to end
+| drop_off_at_location | Location | The specific location at which this assigned ride will end
+| return_assignment_id | string* | If set, the ID of the Assignment which will return the demanding riders to their next location. Typically the next location will be the same one from which the riders departed, but it may be different (example: voter departs from home but is returned after voting to their place of work or school).
+| status | string | One of `scheduled`, `in-progress`, `completed`, or `canceled`. Implementations may prevent certain actions on assignments based on their status at the time (example: an implementation may disallow cancellation of an assigned ride which is `in-progress`).
+
+### Interval Fields
+
+| Name | Type | Description
+|------|--------|------
+| start | datetime | The time at which this interval begins (inclusive)
+| end | datetime | The time at which this interval ends (exclusive)
+
+Intervals determine a discrete and continuous amount of time during which a ride can occur (or more generally, when a thing can occur). Intervals are valid if and only if `start < end`.
+
+### Location Fields
+
+| Name | Type | Description
+|------|--------|------
+| address | [PostalAddress](people.html#postal-addresses) | The physical address for this location
+| identifier | string | A unique identifier within the system for this location. Values may be meaningful to humans ("SC Polling station 1092") or fully opaque database identifiers ("pl2934jk239").
+| availability | Interval[] | If non-empty, availability specifies the times at which this location is available to provide services. For example, a Location representing a polling location might list the hours during which voters are allowed to enter the line to vote.
+
+## Scenarios
+
+The primary motivation for this data model is to describe and accelerate the implementation of applications which provide for matching of drivers and riders, producing assignments. The APIs described here could be used in various ways to build these applications. In one configuration, we can imagine a lower-level routing, dispatch, and scheduling system which is used by API clients to populate their specific rider and driver details, and which provides to those callers a scheduled and on-demand routing and dispatch system. Further, we can imagine that third-parties working as clients to these APIs, but representing neither the specific data owners nor the scheduler's implementation, could build role-specific apps, including mobile apps for drivers or riders, or web-based dispatch and assistance tools for coordinating teams.
```

Comment:
If this is going to be used with people who don’t have smartphones, consider adding something about notifications via SMS and automated calls.
SMS might be prioritized, but we might want to prioritize automated calls since they would work with landlines also e.g. 
For scheduling: "Do you need a ride to the polls? Say yes or press 1, Say no or press 2" 
For reminders: “Your ride to the polls is on the way. Say yes or press 1 to confirm, Say no or press 2 to cancel or reschedule”

Reply:
I added a brief definition of "mobile" to encompass both forms. LMK if you think that is sufficient, or if we need to state more explicitly that the model here is in fact not tied to how communication occurs at all (that's actually my larger position), only that applications have what they need to coordinate that communication as they like.